### ISSUE TITLE
repo name nit: it'd be nice if this were simply w3c/csswg

### ISSUE THREAD

**Issue by hober** [w3c/csswg-drafts#10350]

**Comment by Crissov** [comment]
… or simply `css`.

**Comment by tabatkins** [comment]
Agree, it would be great to shift us to a shorter repo name. I always clone with `git clone gh:w3c/csswg-drafts csswg` anyway, as it's a silly folder name.

**Comment by myakura** [comment]
What will happen to GitHub Pages? Will links to https://w3c.github.io/csswg-drafts/shortname/#hash safely redirect to the new URLs?

**Comment by frivoal** [comment]
@myakura yes.
